### PR TITLE
[IMP] accounting: update doc about Sepa Direct Debit batch payments

### PR DESCRIPTION
**Pull request by jcs-odoo**

### CODE REVIEW

File: accounting/receivables/customer_payments/batch_sdd.rst [hunk 1]
```diff
@@ -1,117 +1,146 @@
...
+   Add link to future **Payment Acquirer** documentation.
 
 Close or revoke a mandate 
 -------------------------
-Direct Debit mandates are closed automatically after their *End Date*. 
-If this field is left blank, then the mandate keeps being *Active* 
-until it is *Closed* or *Revoked*.
 
-.. image:: media/batch_sdd03.png
+Direct Debit mandates are closed automatically after their *End Date*.  If this field is left blank,
```
Comment (kbr-odoo) [inline accounting/receivables/customer_payments/batch_sdd.rst:78]:
```suggestion
Direct Debit mandates are closed automatically after their *End Date*. If this field is left blank,
```

File: accounting/receivables/customer_payments/batch_sdd.rst [hunk 2]
```diff
@@ -1,117 +1,146 @@
...
+then the mandate keeps being *Active* until it is *Closed* or *Revoked*.
```
Comment (kbr-odoo) [inline accounting/receivables/customer_payments/batch_sdd.rst:79]:
```suggestion
the mandate keeps being *Active* until it is *Closed* or *Revoked*.
```

File: accounting/receivables/customer_payments/batch_sdd.rst [hunk 3]
```diff
@@ -1,117 +1,146 @@
...
+
+.. image:: media/batch-sdd-revoke-close.png
    :align: center
+   :alt: Close or Revoke an SDD Mandate in Odoo Accounting
 
-Clicking on **Close** updates the mandate’s end day to the current day. 
-This means that invoices made after the present day won’t be processed 
-automatically with an SDD payment.
+Clicking on **Close** updates the mandate’s end day to the current day. This means that invoices
+made after the present day won’t be processed with an SDD payment.
```
Comment (kbr-odoo) [inline accounting/receivables/customer_payments/batch_sdd.rst:86]:
```suggestion
issued after the present day will not be processed with an SDD payment.
```

File: accounting/receivables/customer_payments/batch_sdd.rst [hunk 4]
```diff
@@ -1,117 +1,146 @@
...
 
 Generate SEPA Direct Debit XML files to submit payments
 -------------------------------------------------------
-To submit your SDD payments to your bank, you will first generate an 
-**XML file** that can be uploaded directly to your bank interface.
 
-The files generated by Odoo follow the SEPA Direct Debit **PAIN.008.001.02** 
-specifications as required by the SEPA customer-to-bank Implementation Guidelines, 
-which ensures compatibility with the banks.
+**XML files** with all SDD payment instructions can be uploaded to your bank institution to process
```
Comment (kbr-odoo) [inline accounting/receivables/customer_payments/batch_sdd.rst:114]:
```suggestion
**XML files** with all SDD payment instructions can be uploaded to your bank account to process
```

File: accounting/receivables/customer_payments/batch_sdd.rst [hunk 5]
```diff
@@ -1,117 +1,146 @@
...
+To generate your XML file for the pending SDD payments, go to the related *bank journal* on your
+*Accounting dashboard*, then click on *Direct Debit Payments to Collect*.
+
+.. image:: media/batch-sdd-collect.png
    :align: center
+   :alt: Start the SDD payments collection in Odoo Accounting
 
-Select all the payments in the list that you want to include in 
-your SDD XML file, then click on *Action* and select *Create Batch Payment*.
+Select all the payments in the list that you want to include in your SDD XML file, then click on
```
Comment (kbr-odoo) [inline accounting/receivables/customer_payments/batch_sdd.rst:129]:
```suggestion
Select all the payments you want to include in your SDD XML file, then click on
```

File: accounting/receivables/customer_payments/batch_sdd.rst [hunk 6]
```diff
@@ -1,117 +1,146 @@
...
+Odoo then takes you to your *Batch Payment*’s form. Click on *Validate* and download the SDD XML
+file.
 
-.. image:: media/batch_sdd06.png
+.. image:: media/batch-sdd-xml.png
    :align: center
+   :alt: Generate an XML file for your SDD payments in Odoo Accounting
 
-Finally, upload this file to your bank to process the payments.
+Finally, upload this file to your bank institution to process the payments.
```
Comment (kbr-odoo) [inline accounting/receivables/customer_payments/batch_sdd.rst:143]:
```suggestion
Finally, upload this file to your bank account to process the payments.
```

File: accounting/receivables/customer_payments/batch_sdd.rst (hunk 4 above)
Comment (jcs-odoo) [reply accounting/receivables/customer_payments/batch_sdd.rst:114]: it's not really uploaded to an account. I'll find another word. Thanks though :)

**Pull request merged**